assets/js/components/higherorder/withdata.test.js

```diff
@@ -131,13 +139,13 @@ describe( 'withData', () => {
 		const dataset = createDataset( TYPE_MODULES, testModule.slug, 'test-datapoint', { dateRange } );
 		const WrappedComponent = withData( TestComponent, [ dataset ] );
 
-		const { container, queryByTestID } = render( <WrappedComponent /> );
+		const { container, queryByTestID } = render( <WrappedComponent />, { setupRegistry } );
 
 		collectModuleData( context );
 
 		expect( queryByTestID( 'test-component' ) ).not.toBeInTheDocument();
 		expect( container.querySelector( '.googlesitekit-cta__title' ) ).toHaveTextContent( 'Test Module activation' );
-		expect( container.querySelector( '.googlesitekit-cta__description' ) ).toHaveTextContent( 'Test Module module needs to be configured' );
+		expect( container.querySelector( '.googlesitekit-cta__description' ) ).toHaveTextContent( 'Test Module module setup needs to be' );
```

Comment:
Let's use the full description here (add "completed"), this is a bit odd otherwise.

Reply:
Updated @felixarntz